### PR TITLE
Index build commands launch

### PR DESCRIPTION
**Pull request by alch-msft**

### CODE REVIEW

File: academic-services/knowledge-exploration-service/how-to-create-index-from-mag.md [hunk 1]
```diff
@@ -2,27 +2,1410 @@
 title: Create index from Microsoft Academic Graph(MAG)
```
Comment (DarrinEide) [inline academic-services/knowledge-exploration-service/how-to-create-index-from-mag.md]: >(MAG) [](start = 49, length = 5)

Remove, we should avoid acronyms in page title metadata

File: academic-services/knowledge-exploration-service/how-to-create-index-from-mag.md [hunk 2]
```diff
@@ -2,27 +2,1410 @@
...
 description: Step-by-step guide for generating MAKES indexes from a MAG release.
 ms.topic: tutorial
-ms.date: 2020-02-10
+ms.date: 2020-02-19
 ---
 
 # Generating MAKES indexes from a MAG release
 
- Step-by-step guide for generating MAKES indexes from a Microsoft Academic Graph(MAG) release. For more information on obtaining a MAG release, visit [Get Microsoft Academic Graph on Azure storage](../graph/get-started-setup-provisioning.md)
+ Step-by-step guide for generating MAKES indexes from a Microsoft Academic Graph(MAG) release. For more information on obtaining a MAG release, visit [Get Microsoft Academic Graph on Azure storage](../graph/get-started-setup-provisioning.md).  In this example we will be building a custom index of all Microsoft publications and associated entities.
```
Comment (DarrinEide) [inline academic-services/knowledge-exploration-service/how-to-create-index-from-mag.md]: Graph(MAG) => Graph (MAG)

File: academic-services/knowledge-exploration-service/how-to-create-index-from-mag.md [hunk 3]
```diff
@@ -2,27 +2,1410 @@
...
 
 ## Prerequisites
 
-- Microsoft Academic Graph(MAG) release
-- Microsoft Academic Knowledge Exploration Service(MAKES) release
-- Azure subscription
+- [Microsoft Academic Graph(MAG) subscription](../graph/get-started-setup-provisioning.md)
```
Comment (DarrinEide) [inline academic-services/knowledge-exploration-service/how-to-create-index-from-mag.md]: >Graph(MAG) [](start = 22, length = 10)

Graph(MAG) => Graph (MAG)

File: academic-services/knowledge-exploration-service/how-to-create-index-from-mag.md [hunk 4]
```diff
@@ -2,27 +2,1410 @@
...
+- [Azure Data Lake Analytics (ADLA) setup with MAG](../graph/get-started-setup-azure-data-lake-analytics.md)
+- [Microsoft Academic Knowledge Service (MAKES) subscription](get-started-setup-provisioning.md)
```
Comment (DarrinEide) [inline academic-services/knowledge-exploration-service/how-to-create-index-from-mag.md]: >Microsoft Academic Knowledge Service [](start = 3, length = 36)

Microsoft Academic Knowledge Service => Microsoft Academic Knowledge Exploration Service

File: academic-services/knowledge-exploration-service/how-to-create-index-from-mag.md [hunk 5]
```diff
@@ -2,27 +2,1410 @@
...
 
 ## Generating MAKES entities using USQL
```
Comment (DarrinEide) [inline academic-services/knowledge-exploration-service/how-to-create-index-from-mag.md]: >Generating MAKES entities using USQL [](start = 3, length = 36)

Generating MAKES indexes using U-SQL

File: academic-services/knowledge-exploration-service/reference-makes-command-line-tool.md [hunk 6]
```diff
@@ -96,6 +96,111 @@ The sku for MAKES API host machines.
...
+
+The MAKES indexing resource configuration file path. Run CreateIndexResource command to generate one.
+
+`--EntitiesUrlPrefix`
+
+The input entities file Url prefix.
+
+`--OutputUrlPrefix`
+
+The output url prefix for writing built index.
```
Comment (RickRogahn) [inline academic-services/knowledge-exploration-service/reference-makes-command-line-tool.md]: "for writing the built index"

File: academic-services/knowledge-exploration-service/reference-makes-command-line-tool.md [hunk 7]
```diff
@@ -96,6 +96,111 @@ The sku for MAKES API host machines.
...
+
+### Optional Parameters
+
+`--IndexPartitionCount`
+
+The number of index partitions to create.
```
Comment (RickRogahn) [inline academic-services/knowledge-exploration-service/reference-makes-command-line-tool.md]: Should we provide guidance somewhere as to the optimal number to create?  Is that in later documentation?

File: academic-services/knowledge-exploration-service/reference-makes-command-line-tool.md [hunk 8]
```diff
@@ -96,6 +96,111 @@ The sku for MAKES API host machines.
...
+
+`--IntersectionMinCount`
+
+The minimum intersections between indexed attribute that indexer should generate pre-calculated results. The higher the count, the more process will be required at run time. The lower the count, the larger index will be generated.
```
Comment (RickRogahn) [inline academic-services/knowledge-exploration-service/reference-makes-command-line-tool.md]: for which the indexer should generate pre-calculated results.

Comment (RickRogahn) [inline academic-services/knowledge-exploration-service/reference-makes-command-line-tool.md]: processing?

Comment (RickRogahn) [inline academic-services/knowledge-exploration-service/reference-makes-command-line-tool.md]: Suggestion: "The lower the count, the larger the generated index will be."

**Comment by DarrinEide** [comment]
Globally please change "USQL" => "U-SQL"

File: academic-services/knowledge-exploration-service/how-to-create-index-from-mag.md [hunk 9]
```diff
@@ -2,27 +2,1410 @@
...
 
+MAKES indexes are generated from specifically formatted MAG data.  In order to create a new index, you will need to generate the entities you would like in the index from MAG.  We will start by running a USQL script on your MAG subscription to generate the data required to build the custom index.  To do this, you will need to go to your Azure Data Lake Analytics (ADLA) service instance and submit a new ALDA job to generate the text files containing academic data.
```
Comment (DarrinEide) [inline academic-services/knowledge-exploration-service/how-to-create-index-from-mag.md]: >MAKES indexes are generated from specifically formatted MAG data.  In order to create a new index, you will need to generate the entities you would like in the index from MAG.  We will start by running a USQL script on your MAG subscription to generate the data required to build the custom index. [](start = 0, length = 297)

MAKES requires data it indexes to be placed in a single JSON file, with each line representing an individual entity. For this example, we will use a U-SQL script to filter and transform MAG data into JSON data which will then be used to create a custom index.

File: academic-services/knowledge-exploration-service/how-to-create-index-from-mag.md [hunk 10]
```diff
@@ -2,27 +2,1410 @@
...
+
+### Define functions to extract MAG data
+
+1. In the [Azure Management Portal](https://portal.azure.com), go to the Azure Data Lake Analytics (ADLA) service that you created, and select **Overview** > **New Job**.
+
+   ![Azure Data Lake Analytics - New job](../graph/media/samples-azure-data-lake-hindex/new-job.png "Azure Data Lake Analytics - New job")
+
+1. Copy code in samples/CreateFunctions.usql under the MAG release and paste into the code block.
```
Comment (DarrinEide) [inline academic-services/knowledge-exploration-service/how-to-create-index-from-mag.md]: >Copy code [](start = 3, length = 9)

Copy the code

Comment (DarrinEide) [inline academic-services/knowledge-exploration-service/how-to-create-index-from-mag.md]: >Copy code in samples/CreateFunctions.usql under the MAG release and paste into the code block. [](start = 3, length = 94)

Copy the code contained in samples/CreateFunctions.usql under the MAG release and paste it into the code window.

File: academic-services/knowledge-exploration-service/how-to-create-index-from-mag.md [hunk 11]
```diff
@@ -2,27 +2,1410 @@
...
+    > [!TIP]
+    > This tutorial uses Microsoft as an organization by default. You can target any organization by finding its NormalizedName in the Microsoft Academic Graph and then changing the value of the affiliationNormalizedNameFilter variable to said name.
+    >
+    > Please note that doing this may impact time estimates for script execution and index generation later on in this tutorial.
+
+5. Provide a **Job name**, change **AUs** to 50, and select **Submit**
+
+   ![Submit GenerateAzureSearchData job](media/how-to-create-index-create-job.png)
+
+The job should finish successfully in about 50 minutes
```
Comment (DarrinEide) [inline academic-services/knowledge-exploration-service/how-to-create-index-from-mag.md]: The job should take approximately 50 minutes to run.

File: academic-services/knowledge-exploration-service/how-to-create-index-from-mag.md [hunk 12]
```diff
@@ -2,27 +2,1410 @@
...
+
 ## Build index using MAKES command line tool
 
+Now that the data has been generated to build your index, we need to create some resources that the index will use and build the index using the kesm.exe tool.
```
Comment (DarrinEide) [inline academic-services/knowledge-exploration-service/how-to-create-index-from-mag.md]: Now that the JSON data has been generated to build the custom index, we use the KESM tool to execute the build.

File: academic-services/knowledge-exploration-service/how-to-create-index-from-mag.md [hunk 13]
```diff
@@ -2,27 +2,1410 @@
...
+    ```cmd
+    kesm CreateIndexResources --IndexResourceName "<IndexResourceName>" --MakesPackage "<MakesPackageLocation>" --MakesIndexResourceConfigFilePath "<MakesIndexResourceConfigFilePath>"
+    ```
+
+1. Replace the command parameters with the values from the table below:
+
+    | Values | Description | Example |
+    |---------|---------|------|
+    |**`<IndexResourceName>`** | The name of the indexing resources you created above. | **makesindexres** |
+    |**`<MakesPackageLocation>`**
```
Comment (DarrinEide) [inline academic-services/knowledge-exploration-service/how-to-create-index-from-mag.md]: >The final step  [](start = 0, length = 15)

Remove

Comment (DarrinEide) [inline academic-services/knowledge-exploration-service/how-to-create-index-from-mag.md]: To build the custom index, submit a job to ADLA using the kesm tool.  The tool uses the Azure resources created in the previous step to build the index and when finished will place the index in your storage account.

File: academic-services/knowledge-exploration-service/how-to-delete-api-instances.md [hunk 14]
```diff
@@ -11,7 +11,8 @@ ms.date: 2020-02-10
 
 ## Prerequisites
 
-- A MAKES API instance deployment
+- [Microsoft Academic Knowledge Service (MAKES) subscription](get-started-setup-provisioning.md)
```
Comment (DarrinEide) [inline academic-services/knowledge-exploration-service/how-to-delete-api-instances.md]: >Microsoft Academic Knowledge Service (MAKES) [](start = 3, length = 44)

Microsoft Academic Knowledge Exploration Service (MAKES)

File: academic-services/knowledge-exploration-service/how-to-delete-api-instances.md [hunk 15]
```diff
@@ -11,7 +11,8 @@ ms.date: 2020-02-10
...
+- [An existing deployment of the Microsoft Academic Knowledge Service (MAKES)](get-started-create-api-instances.md)
```
Comment (DarrinEide) [inline academic-services/knowledge-exploration-service/how-to-delete-api-instances.md]: >Microsoft Academic Knowledge Service (MAKES) [](start = 33, length = 44)

Microsoft Academic Knowledge Exploration Service (MAKES)

**Pull request merged**